packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -2353,7 +2353,7 @@ class _BoxEdge implements Comparable<_BoxEdge> {
 
   @override
   int compareTo(_BoxEdge other) {
-    return (offset - other.offset).sign.toInt();
```

Comment:
I think we should add an assert just so that it still crashes in debug mode. Otherwise, I feel we won't ever figure out what the issue is.

Reply:
line 2333 has the assert that will trigger before this line crashes.